airflow/models/dag.py

```diff
@@ -382,7 +382,7 @@ class DAG(TaskSDKDag, LoggingMixin):
     :param default_view: Specify DAG default view (grid, graph, duration,
                                                    gantt, landing_times), default grid
     :param orientation: Specify DAG orientation in graph view (LR, TB, RL, BT), default LR
-    :param catchup: Perform scheduler catchup (or only run latest)? Defaults to True
+    :param catchup: Perform scheduler catchup (or only run latest)? Defaults to False
```

Comment:
There’s also a docstring in `task_sdk/..../definitions/dag.py` we need to change. (This is duplicated due to the task sdk thing; we should fix this later but not now.)